gen-ai/inference/stable-diffusion-rayserve-gpu/Dockerfile

```diff
@@ -14,13 +14,13 @@ ENV DEBIAN_FRONTEND=non-interactive
 USER $USER
 
 # Install Ray Serve and other Python packages with specific versions
-RUN pip install --no-cache-dir requests torch "diffusers==0.12.1" "transformers==4.25.1"
+RUN pip install --no-cache-dir requests torch "diffusers==0.29.2" "transformers==4.42.4" "accelerate==0.30.1"
 
 # Set a working directory
 WORKDIR /serve_app
 
 # Copy your Ray Serve script into the container
-COPY ray_serve_stablediffusion.py /serve_app/ray_serve_stablediffusion.py
+COPY ray_serve_sd.py /serve_app/ray_serve_sd.py
```

Comment:
Would it be possible to move this serving script as ConfigMap? You can refer to other examples under inference

Reply:
Didn't find an example yet. DM in Slack

Reply:
Check this one as example https://github.com/awslabs/data-on-eks/blob/416447e7ce69ac882c528e6d118e4190cc021789/gen-ai/inference/vllm-rayserve-inf2/vllm-rayserve-deployment.yaml#L30